sdk/python/feast/cli.py

```diff
@@ -155,13 +155,23 @@ def feature_set_create(filename):
 
 
 @feature_set.command("describe")
-@click.argument("name", type=click.STRING)
-def feature_set_describe(name: str):
+@click.option(
+    "--name", "-n", help="Name of feature set", required=True, type=click.STRING
+)
+@click.option(
+    "--project",
+    "-p",
+    help="Project where feature set can be found",
```

Comment:
Perhaps `Project that feature set belongs to`?

Reply:
👌 i'll make that change